lib/content-services/src/lib/document-list/components/document-list.component.spec.ts

```diff
@@ -1907,11 +1907,11 @@ describe('DocumentListComponent rendering', () => {
         const rows = fixture.nativeElement.querySelectorAll('.adf-datatable-body adf-datatable-row');
         expect(rows).toBeDefined();
         expect(rows.length).toBe(3);
-        const cell1 = fixture.nativeElement.querySelector('div[title="Name"][data-automation-id="Name 1"]');
+        const cell1 = fixture.nativeElement.querySelector('td[title="Name"][data-automation-id="Name 1"]');
         expect(cell1.innerText).toBe('Name 1');
-        const cell2 = fixture.nativeElement.querySelector('div[title="Name"][data-automation-id="Name 2"]');
+        const cell2 = fixture.nativeElement.querySelector('td[title="Name"][data-automation-id="Name 2"]');
```

Comment:
the data-automation-id should be a unique value, so not making sense having a long path, could be just `querySelector('[data-automation-id="Name 2"]');`